package.json

```diff
@@ -52,6 +52,7 @@
     "precss": "^3.1.2",
     "prop-types": "^15.6.2",
     "react": "^16.6.3",
+    "react-addons-css-transition-group": "^15.6.2",
```

Comment:
Why not try `popmotion`?

Reply:
Forgot about that, I'll check if it has support for enter/leave transitions and if has I will do it. I think they only make animations. React transition group is to detect changes in the `children` and apply `enter` and `leave` css classes to them.

Reply:
Did with popmotion, but they are JS only animations.

Reply:
Seems simple to use, what do you think? Should we use `popmotion`?

Reply:
Yes